support/appveyor.yml

```diff
@@ -1,22 +1,54 @@
-configuration:
-  - Debug
-  - Release
+configuration: Debug
 
-image: Visual Studio 2017
+clone_depth: 1
+
+platform:
+  - Win32
+  - x64
 
 environment:
   CTEST_OUTPUT_ON_FAILURE: 1
+  MSVC_DEFAULT_OPTIONS: ON
   matrix:
-  - BUILD: msvc
-  - BUILD: msvc
-    PLATFORM: x64
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+      CMAKE_GENERATOR: Visual Studio 12 2013
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      CMAKE_GENERATOR: Visual Studio 14 2015
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      CMAKE_GENERATOR: Visual Studio 15 2017
+
+matrix:
+  allow_failures:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
+      CMAKE_GENERATOR: Visual Studio 12 2013
+
+init:
+  - ps: |
+      If ($env:Platform -Match "x64") {
+        $env:CMAKE_GENERATOR_SUFFIX=" Win64"
+      }
 
 before_build:
-  # Workaround for CMake not wanting sh.exe on PATH for MinGW.
-  - set PATH=%PATH:C:\Program Files\Git\usr\bin;=%
+ - mkdir build
+ - cd build
+    
+ - ps: cmake -G "$($env:CMAKE_GENERATOR)$($env:CMAKE_GENERATOR_SUFFIX)" -DFMT_WERROR=OFF -DFMT_PEDANTIC=ON -DCMAKE_BUILD_TYPE=Debug ..
+
+build:
+  project: C:\projects\fmt\build\fmt.sln
+  parallel: true
+  verbosity: normal
 
-build_script:
-  - python support/appveyor-build.py
```

Comment:
Please keep the Python script, since it's easier to deal with configs in one language and we might want to reintroduce MinGW at some point.

Reply:
I agree that the Python script was much more pleasant to work with. By using the Appveyor's built-in build functionality, we can have build messages and test results reported in a more aesthetically pleasing (tm) way, [as you can see here](https://ci.appveyor.com/project/vitaut/fmt/build/build%201357/job/qugyqem428ehobor). I'll look into incorporating the Python script back, prioritizing bringing it back over the PowerShell trickery.